cookbooks/omnibus-supermarket/files/default/ctl-commands/make-admin.rb

```diff
@@ -0,0 +1,18 @@
+require 'mixlib/shellout'
+
+# supermarket-ctl make_admin username
+add_command 'make-admin', 'Make a Supermarket user an admin', 2 do
+
+  # Find username arg
+  username = ARGV[3]
+
+  # Run rake task
+  command_text = "cd /opt/supermarket/embedded/service/supermarket && sudo RAILS_ENV=\"production\" env PATH=/opt/supermarket/embedded/bin bin/rake user:make_admin user=\"#{username}\""
```

Comment:
Why is it `sudo RAILS_ENV=\"production\" env PATH=/opt/...`? Wouldn't `sudo env RAILS_ENV=\"production\" PATH=/opt/...` make more sense? I suppose if it works it's fine but it looks strange.